docs/local_source_upload.md

```diff
@@ -0,0 +1,44 @@
+Local Source Upload
+-------------------
+
+The purpose of project Shipwright is to transform source code into container images, using known strategies to accomplish this task.
+
+Now, with local source upload, we move Shipwright closer to the developer's inner loop, by allowing streaming local content to the Build Controller, running in a Kubernetes cluster, and creating a container image out of if. Users can try out Shipwright before submitting a pull-request, and rely on the cluster's computing power.
+
+# Usage
+
+The following steps represents the traditional workflow using local source upload:
```

Comment:
The word, "traditional", here is confusing. It implies we're describing the old "pre-local source upload" way of doing things. How about:
```suggestion
To build an image by using the local source upload feature:
```

Reply:
It makes sense. thanks.